frontend/src/employee-frontend/api/unit.ts

```diff
@@ -813,3 +817,22 @@ const toChildDayRows = (
     }))
   )
 })
+
+const deserializeGroupedActualServiceNeeds = (
```

Comment:
Suggestion: don't have two completely different names for one thing: are we deserializing "GroupedActualServiceNeeds" or "UnitServiceNeedInfo"?